pages/calculator.jsx

```diff
@@ -1,15 +1,43 @@
 import { Link } from "react-router-dom";
-
+import { useState } from "react";
 
 const Calculator = () => {
```

Comment:
Since this is a component, I would rename the file to `Calculator`, so with an uppercase letter starting.